<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogMode</class>
 <widget class="QDialog" name="DialogMode">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>300</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 31, 49);
border-color: rgb(255, 255, 255);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="ImageButton" name="Button_Yes">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>174</y>
     <width>140</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>예</string>
   </property>
  </widget>
  <widget class="ImageButton" name="Button_No">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>174</y>
     <width>140</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(192, 191, 188);</string>
   </property>
   <property name="text">
    <string>아니오</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>630</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>회복모드로 변경 하시겠습니까?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Button_Yes</zorder>
  <zorder>Button_No</zorder>
  <zorder>Label_Msg</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageButton</class>
   <extends>QPushButton</extends>
   <header>imagebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
